travel/Html/css/style.css

```diff
@@ -0,0 +1,72 @@
+
+body{
+	font-family: 'Acme', sans-serif;
+}
+p {
+	font-family: sans-serif;
+}
+
+.container{
+	background-color:rgba(0,139,139,.4);	
+	background: -webkit-linear-gradient(top,rgba(0,139,139,1),rgba(0,0,0,0)); /*Safari 5.1-6*/
+	background: -o-linear-gradient(bottom,rgba(0,139,139,1),rgba(0,0,0,0)); /*Opera 11.1-12*/
+	background: -moz-linear-gradient(bottom,rgba(0,139,139,1),rgba(0,0,0,0)); /*Fx 3.6-15*/
+	background: linear-gradient(to bottom, rgba(0,139,139,1), rgba(0,0,0,0)); /*Standard*/
+}
+.head-bkg{
+	background-color: white;
+	background-image: url(http://socialmediaweek.org/wp-content/blogs.dir/1/files/2015/11/Travel_Photography_Blog_SanFrancisco_201111071.jpg);
+	background-repeat: no-repeat;
+	 background-size: 100%;
+}
+
+
+.head-text {	
+	font-family: 'Wendy One', sans-serif;
+	font-size: 7em;
+	color: #FFE4C4;
+	text-shadow: 0px 3px 0px #000000,
+    3px 14px 10px rgba(0,0,0,0.15),
+    6px 24px 2px rgba(0,0,0,0.1),
+    9px 34px 30px rgba(0,0,0,0.1);
+	
+}
+
+.white {
+	color: white;
+}
+.row {
+	margin-top: 15px
+}
+.choose {
+	padding-top: 10px;
+	display:inline-block;   
+    position:relative;
+}
+
+.choose span {
+    display:inline-block;
+    position:absolute;
+    bottom:0px;   
+    width:85%;
+
+    box-sizing:border-box;
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box; 
+    color:#FFF;
+    font-size:20px; 
+    background-color:rgba(0,0,0,.4);	
+	background: -webkit-linear-gradient(left,rgba(0,0,0,1),rgba(0,0,0,0)); /*Safari 5.1-6*/
+	background: -o-linear-gradient(right,rgba(0,0,0,1),rgba(0,0,0,0)); /*Opera 11.1-12*/
```

Comment:
Use just two spaces as it is done in `travel_platform/test_full_landing/css/style.css`